.github/workflows/conformance-k8s-kind-network-policies.yaml

```diff
@@ -28,8 +28,8 @@ env:
   cilium_cli_version: v0.15.11
   cilium_cli_ci_version:
   CILIUM_CLI_MODE: helm
-  # renovate: datasource=docker depName=kindest/node
-  k8s_version: v1.28.0
+  # renovate: datasource=docker depName=quay.io/cilium/kindest-node
```

Comment:
Oh, I kinda assumed we would just leave this with kindest/node and it would use the cilium fork until the next kind release. But maybe that's not the case with renovate? Does renovate downgrade versions?